common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +33,16 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            # Will throw UnicodeDecodeError incase when there are
+            # latin1 encoded characters in string.
+            # Example {'string': '\xd3 \xe9 \xf1'}
+            for key in event.iterkeys():
+                if isinstance(event[key], str):
+                    event[key] = event[key].decode('latin1')
```

Comment:
Here we using latin1 because json.dumps can't handle latin characters by  default.